.github/pull_request_template.md

```diff
@@ -0,0 +1,3 @@
+<!-- Please set the title of this PR according to https://www.conventionalcommits.org/en/v1.0.0/#summary, and delete this line and similar ones -->
+
+<!-- What does this do, and why do we need it? -->
```

Comment:
```suggestion
<!-- What does this PR do, and why do we need it? -->
```

Reply:
It wasn't clear to me, I thought that line was some sort of test from your side 😸

Reply:
It is "inspired" (copied) from the Kubernetes PR template.